### ISSUE TITLE
This is Awesome! 

### ISSUE THREAD

**Issue by gholzmacher** [des-des/ra-postgraphile-client#1]
Hi this is great! I have spent the last three days trying to use postgREST and reworking the unsupported provider from AOR on RA.  What does your timeline look like for version 1.0 ?

**Comment by des-des** [comment]
Hi @gholzmacher sorry for the delay getting back to you.

What I have now on my machine seems to be mostly there so Ill try to get a v1 up this weekend!

Thanks for being nice!

**Comment by gholzmacher** [comment]
Thank You so much! I am very anxious to start building!

I am in awe of the postgrest/prographile concept! It makes so much sense to
only define the data once for tha api! Then combined with react admin this
is getting close to lego style application building! Its nothing short of
amazing!

**Comment by des-des** [comment]
Yeah hooking postgraphile up to react admin seemed like a pretty logic step.

I have released a V1 now pushed up some more stuff. It is a long way from complete and needs tests and more but it least now ill keep to semver. I am using this in production but there are some bits that are not quite working yet...
